src/scripts.js

```diff
@@ -99,6 +100,10 @@ function dropYear(dates) {
     return userData.find(user => user.id === id).name;
   }
 
+  function findUserCard(id) {
```

Comment:
What is this function doing? I don't see it called anywhere in the functions you wrote below. It looks like it is returning the users data object. Also, it looks like this branch is not up to date with the fetch/data merge we did on Friday, which has all of the data being taken and used from the API rather than being imported and we definitely do not want to lose that when merging this in - So you will need to account for that when fixing the merge conflicts on your local side.

Reply:
I was using it before I refractored the section. The branch should be good to merge on github now. I can't merge my own stuff and a large part of this project is using github to merge even when you have a conflict. You won't lose any work if you look at what you are merging into the master. We can do this.